<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateProfile</class>
 <widget class="QDialog" name="CreateProfile">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>374</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>443</width>
    <height>374</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>443</width>
    <height>374</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Generate Signature</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>334</x>
     <y>35</y>
     <width>98</width>
     <height>26</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Additional Settings</string>
   </property>
   <property name="text">
    <string>Options</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/preferences-other.png</normaloff>:/icons/icons/preferences-other.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="lblConsoleOutput">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>183</y>
     <width>103</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Console Output:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="packageList">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>146</x>
     <y>110</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="toolTip">
    <string>Install these packages separated by comma</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblPackageList">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>110</y>
     <width>128</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'DejaVu Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Install these packages&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:7pt;&quot;&gt;separated by comma&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSaveProfile">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>153</y>
     <width>122</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Signature As </string>
   </property>
  </widget>
  <widget class="QCheckBox" name="installPackagesRadioBox">
   <property name="geometry">
    <rect>
     <x>122</x>
     <y>63</y>
     <width>127</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Select to install specified packages</string>
   </property>
   <property name="text">
    <string>Install Packages</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="updateCheckBox">
   <property name="geometry">
    <rect>
     <x>122</x>
     <y>13</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Select for APT's Database Update</string>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="profileFilePath">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>146</x>
     <y>148</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="toolTip">
    <string>File to save signature as</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browseFilePathButton">
   <property name="geometry">
    <rect>
     <x>334</x>
     <y>148</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>91</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblInstallType">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>38</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Installation Type</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="upgradePackagesRadioBox">
   <property name="geometry">
    <rect>
     <x>122</x>
     <y>37</y>
     <width>142</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Select for Packages that need to be upgraded</string>
   </property>
   <property name="text">
    <string>Upgrade Packages</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>204</y>
     <width>421</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="consoleOutputHolder">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="createProfileButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Press to Generate Signature</string>
        </property>
        <property name="text">
         <string>Create</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/icons/dialog-ok-apply.png</normaloff>:/icons/icons/dialog-ok-apply.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Press to Cancel operation</string>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/icons/dialog-cancel.png</normaloff>:/icons/icons/dialog-cancel.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
